pkgs/applications/virtualization/virt-viewer/default.nix

```diff
@@ -26,7 +26,7 @@
 , python3
 , shared-mime-info
 # https://gitlab.com/virt-viewer/virt-viewer/-/issues/88
-, spice-gtk_libsoup2
+, spice-gtk
```

Comment:
Would be nice to mention in the commit message that this only works thanks to libgovirt being disabled.

Reply:
The libgovirt update includes soup3. In fact, it doesn't work with soup2.